libraries/core-react/src/Slider/Slider.tsx

```diff
@@ -227,27 +238,28 @@ export const Slider = forwardRef<HTMLDivElement, Props>(function EdsSlider(
       onChange(event, [changedValue])
     }
   }
-  const handleKeyUp = (event) => {
+  const handleKeyUp = (event: KeyboardEvent) => {
     if (event.keyCode === 37 || event.keyCode === 39) {
       handleCommitedValue(event)
     }
   }
 
-  const handleCommitedValue = (event) => {
+  const handleCommitedValue = (event: MouseEvent | KeyboardEvent) => {
```

Comment:
Why did this work previously? 🤔

Reply:
Did you try `React.MouseEvent` and `React.KeyboardEvent`? The types you have her are the global ones...

Reply:
They are importet from React @mimarz